extension/src/plots/webview/contract.ts

```diff
@@ -151,11 +156,22 @@ export interface TemplatePlotsData {
   smoothPlotValues: SmoothPlotValues
 }
 
+export type ComparisonPlotBoundingBox = {
```

Comment:
@mattseddon, I've updated this type to not be plural, but, apologies, but I cannot, for the life of me, recall the other suggestion you made about the types during our mob programming session 🤦‍♀️. I think it was about `ComparisonBoundingBoxClasses`? 

I'll be sure to remember to bring my notebook for the next meeting 😅

Reply:
Great points! I've updated the types to be `ComparisonClassDetails` and `ComparisonPlotClass`.